### PR TITLE
Playlists

### PR DESCRIPTION
**Pull request by rahulyadav139**
Demo: https://statuesque-creponne-891db8.netlify.app/

### CODE REVIEW

File: src/pages/SinglePlaylist.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+          <h1>My Playlist</h1>
+          <button className="btn error rounded-edge">Delete Playlist</button>
+        </div>
+        <div className="videos-container">
+          {Array.from({ length: 4 }).map(el => (
+            <VideoCard dismissBtn={true} />
+          ))}
+        </div>
+      </main>
+    </Fragment>
```
Comment (ivinayakg) [inline src/pages/SinglePlaylist.js:20]: as you are not using any of the features of fragments instead of using this component just use empty tag "<></>"

**Review by ivinayakg — APPROVED** [review]
clean work, just one suggestion if you have a page file and a CSS file corresponding to it, then just create a folder and add them to the pages folder

File: src/pages/SinglePlaylist.js (hunk 1 above)
Comment (rahulyadav139) [reply src/pages/SinglePlaylist.js:20]: Thank you for pointing that out. I'll keep it in mind.

**Review by j24m — APPROVED** [review]
Code looks good to me. Amazing work on the UI.

**Pull request merged**